### PR TITLE
Run the code formatter on Macro

### PR DESCRIPTION
**Pull request by obahareth**

### CODE REVIEW

File: lib/elixir/lib/macro.ex [hunk 1]
```diff
@@ -624,7 +627,11 @@ defmodule Macro do
   # Splat when
   def to_string({:when, _, args} = ast, fun) do
     {left, right} = split_last(args)
-    fun.(ast, "(" <> Enum.map_join(left, ", ", &to_string(&1, fun)) <> ") when " <> to_string(right, fun))
+
+    fun.(
+      ast,
+      "(" <> Enum.map_join(left, ", ", &to_string(&1, fun)) <> ") when " <> to_string(right, fun)
```
Comment (josevalim) [inline lib/elixir/lib/macro.ex]: Can you please compute the result of the second argument on the previous line and call `fun.(ast, string)`? You will need to do similar changes in many clauses below. Thanks!

**Comment by obahareth** [comment]
Alright, I’ll take care of it today.

**Comment by obahareth** [comment]
@josevalim I went through the file a number of times and changed all the ones I found, I hope that's all of them.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**